package.json

```diff
@@ -25,9 +25,9 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "jest -c jest.config.js --watch",
```

Comment:
react-script does not work nicely when trying to define your own jest config, hence the change